src/parser/formatting-evaluator-module.ts

```diff
@@ -55,22 +58,24 @@ export class FormattingEvaluatorModule implements Module {
         const { formatting } = this._getFormattingScore(comment);
         const multiplierFactor = this._multipliers?.[comment.type] ?? { wordValue: 0, formattingMultiplier: 0 };
         const formattingTotal = formatting
-          ? Object.values(formatting).reduce(
-              (acc, curr) =>
-                acc.add(
-                  new Decimal(curr.score)
+          ? Object.values(formatting).reduce((acc, curr) => {
+              let sum = new Decimal(0);
+              for (const symbol of Object.keys(curr.symbols)) {
+                sum = sum.add(
+                  new Decimal(curr.symbols[symbol].count)
+                    .mul(curr.symbols[symbol].multiplier)
                     .mul(multiplierFactor.formattingMultiplier)
-                    .mul(curr.count)
-                    .mul(multiplierFactor.wordValue)
-                ),
-              new Decimal(0)
-            )
+                    .mul(curr.score)
```

Comment:
In my memory score was basically a reward for each tag, for example if there are 5 p tags and the score in the config is set to 5, then the reward is 25.

If I understand correctly now symbols/regex is counted inside the tag and multiplied by the multiplier for the symbol and then multiplied by formatting multiplier which is a multiplier which is same for all symbols and tags and then score is basically like a multiplier which is same for all symbols inside the tag?

Reply:
@whilefoo yes that's pretty much what is happening there. Shall it be changed? If so I can take care of it in a separate PR.